doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,202 @@ You should see something like
+    Please direct any questions, comments, or issues to the Amazon Ray Team at `<https://github.com/amzn/amazon-ray/issues/new/choose>`_.
+
+.. list-table:: Unified CloudWatch Agent Images
+
+    * - Base AMI
+      - AMI ID
+      - Region
+      - Unified CloudWatch Agent Version
+    * - AWS Deep Learning AMI (Ubuntu 18.04, 64-bit)
+      - ami-069f2811478f86c20
+      - us-east-1
+      - v1.247348.0b251302
+    * - AWS Deep Learning AMI (Ubuntu 18.04, 64-bit)
+      - ami-058cc0932940c2b8b
+      - us-east-2
+      - v1.247348.0b251302
+    * - AWS Deep Learning AMI (Ubuntu 18.04, 64-bit)
+      - ami-044f95c9ef12883ef
+      - us-west-1
+      - v1.247348.0b251302
+    * - AWS Deep Learning AMI (Ubuntu 18.04, 64-bit)
+      - ami-0d88d9cbe28fac870
+      - us-west-2
+      - v1.247348.0b251302
+
+.. note::
+
+    Using Amazon CloudWatch will incur charges, please refer to `CloudWatch pricing <https://aws.amazon.com/cloudwatch/pricing/>`_ for details.
+
+Basic Setup:
+~~~~~~~~~~~~
+
+1. Create a minimal YAML cluster config named ``cloudwatch-basic.yaml`` with the following contents:
+
+.. code-block:: yaml
+
+    provider:
+        type: aws
+        region: us-west-2
+        availability_zone: us-west-2a
+        # Start by defining a `cloudwatch` section to enable CloudWatch integration with your Ray cluster.
+        cloudwatch:
+            agent:
+                # Path to Unified CloudWatch Agent config file
+                config: "cloudwatch/example-cloudwatch-agent-config.json"
+            dashboard:
+                # CloudWatch Dashboard name
+                name: "example-dashboard-name"
+                # Path to the CloudWatch Dashboard config file
+                config: "cloudwatch/example-cloudwatch-dashboard-config.json"
+
+    auth:
+        ssh_user: ubuntu
+
+    available_node_types:
+        ray.head.default:
+            node_config:
+            InstanceType: c5a.large
+            ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+            resources: {}
+        ray.worker.default:
+            node_config:
+                InstanceType: c5a.large
+                ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+                IamInstanceProfile:
+                    Name: ray-autoscaler-cloudwatch-v1
+            resources: {}
+            min_workers: 0
+
+2. Download example CloudWatch agent and dashboard configuration files:
+
+   First, create a ``cloudwatch`` directory in the same directory as ``cloudwatch-basic.yaml``.
+   Then, download the `CloudWatch agent config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json>`_ and `CloudWatch dashboard config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json>`_ to the cloudwatch directory.
```

Comment:
Can we also provide some shell commands to download these?

Reply:
Sure, added command to download JSON config files.